roles/openshift_control_plane/tasks/main.yml

```diff
@@ -65,6 +65,12 @@
     dest: "{{ openshift_master_scheduler_conf }}"
     backup: true
 
+- name: Generate encryption config
+  copy:
+    content: "{{ openshift_encryption_config }}"
+    dest: "{{ openshift.common.config_base }}/master/encryption-config.yaml"
+  when: api_server_args is defined and api_server_args['experimental-encryption-provider-config'] is defined and openshift_encryption_config is defined 
```

Comment:
api_server_args does not appear to be a valid variable name.  In any case, we don't need to check for 'api_server_args' in any form, we can just put this file if openshfit_encryption_config is defined, master's config will be handled independently of this task.

Reply:
I found it was osm_api_server_args as in the above posting this works with 3.10.45 recently.